subsys/bluetooth/audio/unicast_client.c

```diff
@@ -1809,9 +1826,7 @@ static uint8_t unicast_client_cp_discover_func(struct bt_conn *conn,
 	LOG_DBG("conn %p attr %p handle 0x%04x", conn, attr, chrc->value_handle);
 
 	params->err = 0;
```

Comment:
is it still necessary to clear params->err here?

Reply:
Nope :) Good catch